week01/Merge_Sorted_Array.py

```diff
@@ -0,0 +1,11 @@
+class Solution:
+    # TC: O(nlogn), MC: O(1)
+    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
+        """
+        Do not return anything, modify nums1 in-place instead.
+        """
+        for i in range(n):
+            nums1[m] = nums2[i]
+            m+=1
+        
+        nums1.sort()
```

Comment:
Not acceptable. Not at all. If two arrays are already sorted, they can be merged in a very simple manner similar to merge sort. And the complexity is only O(N).